flink-tsfile/src/main/java/org/apache/iotdb/flink/tsfile/util/TSFileConfigUtil.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.iotdb.flink.tsfile.util;
+
+import org.apache.iotdb.tsfile.common.conf.TSFileConfig;
+import org.apache.iotdb.tsfile.common.conf.TSFileDescriptor;
+
+/**
+ * Utils of setting global TSFileConfig.
+ */
+public class TSFileConfigUtil {
+	
+	public static void setGlobalTSFileConfig(TSFileConfig config) {
+		TSFileConfig globalConfig = TSFileDescriptor.getInstance().getConfig();
```

Comment:
Hi, this may be hard to maintain... how about let the user directly use TSFileConfig as follows if they want to change some configuration:

TSFileConfig config = TsFileDescriptor.getInstance().getConfig();
config.setXXX();

Reply:
Yes this seems not a good solution.. But users could not set the TSFileConfig of TsFile flink connector directly because the connector is running on flink cluster, which may be on other machines. So in our design the connector object (i.e. `TsFileInputFormat`) could store a TSFileConfig object, which will be serialized and upload to remote flink cluster and set as the global TSFileConfig when the connector is initializing on the remote flink cluster. 
We certainly have other solutions, e.g. expose a setter on `TsFileDescriptor` or just using reflection. I think this needs to be discussed so keep the original implementation when creating this PR.

Reply:
I know, thank you!